test/test_force.js

```diff
@@ -86,20 +86,36 @@ function main(args) {
         }
     }
     else {
-        for (var cliName in SDK.forceclis) {
-            var cli = SDK.forceclis[cliName];
-            if (cli.platforms.some(p=>chosenOperatingSystems.indexOf(p)>=0)
-                && cli.appTypes.some(a=>chosenAppTypes.indexOf(a)>=0)) {
-
-                if (testProduction) {
-                    // Install forcexxx package
-                    installPublishedForceCli(tmpDir, cli);
-                }
-                else {
-                    // Create forcexxx packages needed
-                    createDeployForcePackage(tmpDir, cli);
+        var forceClis= [];
+
+        if (testingWithClis) {
```

Comment:
The clis to create/deploy are either:
- the ones chosen by the user with the cli parameter
- the ones needed for the chosen os or apptype